internal/commands/push/command.go

```diff
@@ -407,10 +407,10 @@ func deployDraftAndWait(ui terminal.UI, realmClient realm.Client, to to, draftID
 		for deployment.Status == realm.DeploymentStatusCreated || deployment.Status == realm.DeploymentStatusPending {
 			time.Sleep(time.Second)
 
-			deployment, err = realmClient.Deployment(to.GroupID, to.AppID, deployment.ID)
+			deployment, err = realmClient.Deployment(remote.GroupID, remote.AppID, deployment.ID)
 			if err != nil {
-				if e := realmClient.DiscardDraft(to.GroupID, to.AppID, draftID); e != nil {
-					ui.Print(terminal.NewWarningLog("Failed to discard the draft created for your deployment"))
+				if e := realmClient.DiscardDraft(remote.GroupID, remote.AppID, draftID); e != nil {
+					ui.Print(terminal.NewWarningLog("Failed local discard the draft created for your deployment"))
```

Comment:
find/replace shenanigans...

Reply:
Nice catch! always need a second pair of eyes on this stuff lol